### ISSUE TITLE
[DEV] diskfull installation, the pre-install log can't be saved in CN for PowerKVM

### ISSUE THREAD

**Issue by penguhyang** [xcat2/xcat-core#1093]
After the provision, in the CN node.
The output looks like:
```
sleep 35
/opt/xcat/xcatinfo generated
trying to download postscripts from http://10.3.17.14/install/postscripts/
postscripts downloaded successfully
trying to download precreated mypostscript file http://10.3.17.14/tftpboot/mypostscripts/mypo
stscript.c910f03c17k15
failed to download precreated mypostscript, trying to generate with getpostscript.awk
generate mypostscript file successfully
/xcatpost/mypostscript.post generated
/etc/init.d/xcatpostinit1 generated
service xcatpostinit1 enabled
/opt/xcat/xcatinstallpost generated
/opt/xcat/xcatdsklspost generated
running mypostscript
Thu May 12 01:11:43 EDT 2016 Running postscript: syslog
Thu May 12 01:11:43 EDT 2016 postscript syslog return with 0
Thu May 12 01:11:43 EDT 2016 Running postscript: remoteshell
Thu May 12 01:11:47 EDT 2016 postscript remoteshell return with 0
Thu May 12 01:11:47 EDT 2016 Running postscript: syncfiles
Thu May 12 01:11:47 EDT 2016 postscript syncfiles return with 0
mypostscript returned
/boot/grub/grub.conf updated
finished node installation, reporting status...
running /xcatpost/mypostscript.post
Thu May 12 01:13:53 EDT 2016 Running postscript: otherpkgs
Thu May 12 01:13:53 EDT 2016 postscript otherpkgs return with 0
node booted, reporting status...
/xcatpost/mypostscript.post return
service xcatpostinit1 disabled
```
It lacks of some getinstdisk script output from pre-install script.

**Comment by whowutwut** [comment]
@penguhyang  can you assign this to a milestone so it's not lost

**Comment by penguhyang** [comment]
@whowutwut Thanks for review.
This issue will be fixed in 2.12.1
